### ISSUE TITLE
[Bug] The game crash when try to break the Water Saw

### ISSUE THREAD

**Issue by Fireztonez** [vadis365/PrimalTech#13]
Hello, 
Hello, I use your mod in my modpack and I love it, thank you very much !!
For example, every time I try to break the Water Saw with an ax, my game crashes completely.

Minecraft: 1.12.2
Forge: 14.23.2.2632
Primal Tech: 0.2.98

Fireztonez

**Comment by codetaylor** [comment]
I can confirm.

Steps to reproduce:
1. place water saw
2. equip bone axe
3. break water saw with bone axe

Here is my full crash log: https://gist.github.com/codetaylor/197ad2ff30f54ee3b1ed88178e08f1f9

Here is an excerpt:
```
java.lang.RuntimeException: Missing particle!
	at net.minecraft.client.renderer.block.model.SimpleBakedModel$Builder.func_177645_b(SourceFile:146)
	at net.minecraftforge.client.ForgeHooksClient.getDamageModel(ForgeHooksClient.java:657)
	at net.minecraft.client.renderer.BlockRendererDispatcher.func_175020_a(BlockRendererDispatcher.java:46)
	at codechicken.lib.render.block.CCBlockRendererDispatcher.func_175020_a(CCBlockRendererDispatcher.java:39)
...
```

Minecraft: 1.12.2
Forge: 14.23.2.2611
Primal Tech: 0.2.99

**Comment by vadis365** [comment]
Fixed in Primal 0.3.0 - release on the way.

**Comment by Fireztonez** [comment]
Thank's for the fix vadis365
